python/hail/tests/test_methods.py

```diff
@@ -388,6 +388,19 @@ def hail_calculation(ds):
 
         self.assertTrue(np.allclose(manual, rrm))
 
+    def test_hwe_normalized_pca(self):
+        dataset = hl.balding_nichols_model(3, 100, 100)
+        eigenvalues, scores, loadings = hl.hwe_normalized_pca(dataset.GT, k=2, compute_loadings=True)
+
+        self.assertEqual(len(eigenvalues), 2)
+        self.assertTrue(isinstance(scores, hl.Table))
+        self.assertEqual(scores.count(), 100)
+        self.assertTrue(isinstance(loadings, hl.Table))
+        self.assertEqual(loadings.count(), 100)
```

Comment:
you can't depend on 100, we drop variants if they don't vary since they can't be normalized. But I'm not sure why you're only getting 49.